.goreleaser.yml

```diff
@@ -89,9 +89,9 @@ nfpms:
           amd64: x86_64
           arm64: aarch64
           armhf: armv7hl
-        file_name_template: "influxdb2-{{ .Version }}.{{ .Arch }}"
+        file_name_template: "influxdb2-{{ .Env.VERSION }}.{{ .Arch }}"
```

Comment:
Do we want to use a fixed string like `2.0-nightly` for filenames instead of the full version? That way there's a URL that stays stable as we push more releases.

Reply:
That's what I had originally, but then I wondered about the difficulty in identifying/debugging nightly builds. Maybe we don't care that much about that? We could do post-hoc management of the "latest" artifacts, basically copy them to the stable URL. But you're probably right that it's not worth the trouble. I'll change it.

Reply:
Fixed (in the variable injection in the Circle command)